### ISSUE TITLE
   image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB) cv2.error: OpenCV(3.4.4) /io/opencv/modules/imgproc/src/color.cpp:181: error: (-215:Assertion failed) !_src.empty() in function 'cvtColor'

### ISSUE THREAD

**Issue by ArjunCRPSM** [markjay4k/YOLO-series#32]
please help!!!!!!!!!!!!!!!!!!!!

**Comment by Nosherwan-Akram** [comment]
@markjay4k Can you look into this? Having the same issue here.
So basically when I am annotating images I get several of the images annotated but after that I run into this issue for the next upcoming image that needs to be annotated.

**Comment by Nosherwan-Akram** [comment]
Figured this out the rename function ended up creating image 00071.png as a folder instead of an image. I deleted this folder and ran the annotation script again. This solved the issue for me. What I came to understand was that this specific error is the cv2's way of telling that it can not process the image path given and the issues lies there so I suggest using the print(image.path) in your script to break down where the actual error is @ArjunCRPSM

**Comment by mpky** [comment]
Yeah cv2 won't throw an error when reading in an image even if an image actually isn't there. Probably why the error did appear once it went through cvtColor.
